### ISSUE TITLE
i2c driver not installed when scanning

### ISSUE THREAD

**Issue by nkolban** [nkolban/ESP32_Explorer#13]
When I issue an I2C scan, we get errors in the log.  See the following pastebin:

https://pastebin.com/jZe2YeHp

**Comment by chegewara** [comment]
Yes, its small design flaw. At the moment before you can scan you have to open I2C port. To open I2C port you need to switch to command tab and setup which pins and what speed port you want.
